File: app/src/main/java/com/example/rider/SubmitSuccessActivity.java
package com.example.rider;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class SubmitSuccessActivity extends AppCompatActivity {
Button submitHomeBtn;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_submit_success);

        submitHomeBtn =  findViewById(R.id.submitHomeBtn_id);
        submitHomeBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(SubmitSuccessActivity.this, StudentSideNavBar.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/rider/ui/SubmitSuccessActivity.kt
package com.example.rider.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.rider.databinding.ActivitySubmitSuccessBinding

class SubmitSuccessActivity : AppCompatActivity() {
    private var binding: ActivitySubmitSuccessBinding? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivitySubmitSuccessBinding.inflate(layoutInflater)
        setContentView(binding!!.root)

        binding?.submitHomeBtnId?.setOnClickListener { _ ->
//            Intent(this@SubmitSuccessActivity, StudentSideNavBarActivity::class.java).also {
//                startActivity(it)
//            }
            finish()
        }
    }
}